src/components/study-pane.js

```diff
@@ -322,6 +327,13 @@ const StudyPane = (props) => {
         }
     };
 
+    const stopComputation = (runnable) => {
+        if (runnable === Runnable.SECURITY_ANALYSIS) {
+            stopSecurityAnalysis(studyName, userId);
+            setComputationStopped(!computationStopped);
```

Comment:
See comment about status `stopped`

Reply:
does this really need to be a toggle ? why not setComputationStopped(true) ?